ext/bcmath/libbcmath/src/str2num.c

```diff
@@ -57,77 +60,91 @@ bool bc_str2num(bc_num *num, char *str, size_t scale)
 	while (*ptr == '0') {
 		ptr++;
 	}
+	integer_ptr = ptr;
```

Comment:
I'll only put this comment once as it applies to some other places too.
When refactoring code, you should merge the declaration and the assignment where possible.
I.e. make this `char *integer_ptr = ptr;`. Or even `const char *integer_ptr = ptr;`.
The reason to merge them is so it becomes clearer what the actual scope of the variable is, so it's not accidentally used when it should not be used (yet).